Reqnroll.VisualStudio/UI/ViewModels/AddNewReqnrollProjectViewModel.cs

```diff
@@ -5,16 +5,17 @@ public class AddNewReqnrollProjectViewModel : INotifyPropertyChanged
 {
     private const string MsTest = "MsTest";
     private const string Net8 = "net8.0";
+    private const string Net9 = "net9.0";
 
 #if DEBUG
     public static AddNewReqnrollProjectViewModel DesignData = new()
     {
-        DotNetFramework = Net8,
+        DotNetFramework = Net9,
         UnitTestFramework = MsTest,
         FluentAssertionsIncluded = false
     };
 #endif
-    private string _dotNetFramework = Net8;
+    private string _dotNetFramework = Net9;
```

Comment:
I think this should also be reverted to Net8 to keep consistency

Reply:
I'm curious, consistent with what? Do we use net8 as default somewhere else?

Reply:
With `IsSelected=True` here: https://github.com/reqnroll/Reqnroll.VisualStudio/pull/113/files#diff-ab1aa1ab7580a9c8e9242e0a1db8e893909381cb5af52e3c0782300b647b345aR233

We discussed with @clrudolphi that since the std class lib project also offers .NET 8.0 as default, we should do the same.